app/Http/Controllers/User/HighlightsController.php

```diff
@@ -133,6 +133,9 @@ public function index(Request $request, $user_id)
         $color        = checkParam('color');
         $limit        = (int) (checkParam('limit') ?? 25);
 
+        $sort_by_book = $sort_by === 'book';
+        $order_by = $sort_by_book ? DB::raw('books.protestant_order, user_highlights.chapter, user_highlights.verse_start') : 'user_highlights.' . $sort_by;
```

Comment:
I think we have a misunderstanding here maybe.  You're using `protestant` ordering by default.  I thought we were going to look at the specific bible for the highlight and get the order integer the specific bible version/book order and sort by that.

Reply:
me too

Reply:
Done, now is ordered by the versification order of the bible